assets/js/phoenix_live_view/index.js

```diff
@@ -0,0 +1,13 @@
+/*
+================================================================================
+Phoenix LiveView JavaScript Client
+================================================================================
+
+See the hexdocs at `https://hexdocs.pm/phoenix_live_view` for documentation.
+
+*/
+
+import LiveSocket from "phoenix_live_view/live_socket"
+export {
+  LiveSocket
+}
```

Comment:
```suggestion
export { default as LiveSocket } from "phoenix_live_view/live_socket"
```

Reply:
I'm OK with this style change but I think it will come down to if this style is what @chrismccord wants.

Reply:
We are not exporting `LiveSocket` as `default`

Reply:
Actually, I take that back. There appears to be some inconsistency in the existing JS. `LiveScocket` is being exported as `default` and as a named export: https://github.com/phoenixframework/phoenix_live_view/blob/master/assets/js/phoenix_live_view.js#L3018

The generated `app.js` from Phoenix assumes named export: https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_assets/app.js#L18

The documentation in Elixir assumed `default` export: https://github.com/phoenixframework/phoenix_live_view/blob/master/lib/phoenix_live_view.ex#L178

but that code snippet is not consistent with the `app.js` code: https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_assets/app.js#L18

I am OK adding the `default` export, I just need @chrismccord to weigh in on what the intention is here. I suspect somewhere along the way this detail got confused and we ended up with both ways.

Reply:
If we want we can deprecate the default export by adding a `console.log` message indicating that `default` is deprecated in favor of the named import, then align all documentation around the named import.

Reply:
@bcardarella that line should export the default export as a named export.

Basically the same you wrote in one statement.

Reply:
Ah I see, that's interesting. I didn't know you could do both at once. Let me get feedback from @chrismccord on the `default` vs named export preference. If a change is going to be made now would be the time. If no change will be made I'll happily change to your suggestion.

Reply:
I spoke with @chrismccord and we're going to just go with named export and not have a `default` export.